testdriver/ddtargs.py

```diff
@@ -2,7 +2,7 @@
 
 # Args expected with examples
 #  test_type: coll_shift, decimal_fmt, etc.
-#  exec: nodejs, rust, cpp, java, custom
```

Comment:
Note: it's okay for these identifiers to differ from the command line arguments. It's also okay to change them. But, the Node.js binary being named "node" does not imply that we need to change the name here.